xml/System.Threading/Timer.xml

```diff
@@ -727,7 +727,7 @@ Sub TimerCallback(state As Object)
 ## Remarks  
  The callback method is invoked once after `dueTime` elapses, and thereafter each time the time interval specified by `period` elapses.  
   
- If `dueTime` is zero (0), the callback method is invoked immediately. If `dueTime` is negative one (-1) milliseconds, the callback method is never invoked; the timer is disabled, but can be re-enabled by calling <xref:System.Threading.Timer.Change%2A> and specifying a positive value for `dueTime`.  
+ If `dueTime` is zero (0), the callback method is invoked immediately. If `dueTime` is <see cref="F:System.Threading.Timeout.Infinite" />, the callback method is never invoked; the timer is disabled, but can be re-enabled by calling <xref:System.Threading.Timer.Change%2A> and specifying a positive value for `dueTime`.  
```

Comment:
The dueTime in this overload is a TimeSpan, not an int.  If it's changing to be something on Timeout, it should be InfiniteTimeSpan, not Infinite.